src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4954,13 +4965,30 @@ public override bool PreVisit(CallNode node)
                         _txb.AddVolatileVariables(args[i], volatileVariables);
                     }
 
+                    var isIdentifier = _features.HasFlag(Features.SupportIdentifiers) && maybeFunc.IsIdentifier(i);
```

Comment:
>isIdentifier [](http://example.com/codeflow?start=24&length=12)

if this is true, we just need to ensure the arg is a FirstNameNode and not do anything else. 

The rest of the work is then just in CheckInvocation and IR. 

And then CheckInvocation will verify that it's a "valid" identifier. 
For example, for DropCOlumns, it will ensure the identifier is a field on arg0. 
For AddColumns, probably ensures the identifier is not  yet on Arg0 (double check)